src/Microsoft.ML.StandardTrainers/Standard/SdcaBinary.cs

```diff
@@ -965,6 +966,7 @@ private void InitializeConvergenceMetrics(out string[] names, out Double[] initi
                 // Iterates through data to compute loss function.
                 while (cursor.MoveNext())
                 {
+                    Host.CheckAlive();
```

Comment:
Is it a cheap operation? We call this function on every data point.

Reply:
It’s  a two line operation but if you haven’t read the issue you should.

Reply:
There is no description about this function in your issue....